webpack.development.config.js

```diff
@@ -35,7 +35,7 @@ const config = {
         extensions: [ '.tsx', '.ts', '.js' ]
     },
     output: {
-        filename: 'ui-offline.js',
+        filename: 'flow-offline.js',
```

Comment:
Currently offline isn't a part of any bundles, and people have to manually add it to their Players.
Renaming this will mean that people who added offline previously to their Player will not receive the latest updates from flow-offline.js.
Is that okay?

Reply:
As discussed, this should only be the case if they're manually adding the script tag and that should only be necessary when fixing their version and not using loader.js.